packages/vuetify/src/composables/stack.ts

```diff
@@ -14,8 +14,13 @@ interface StackProvide {
 
 const globalStack = reactive<[uid: number, zIndex: number][]>([])
 
-export function useStack (isActive: Readonly<Ref<boolean>>, zIndex: Readonly<Ref<string | number>>) {
+export function useStack (
+  isActive: Readonly<Ref<boolean>>,
+  zIndex: Readonly<Ref<string | number>>,
+  disableGlobalStack: Readonly<Ref<boolean>>
```

Comment:
In order to guarantee correct behavior I think this should be just a boolean, not a ref containing a boolean.

Reply:
Could you elaborate on why behavior would be incorrect?

Reply:
Well, it's an edge case but with the current implementation, if someone changed the `disableGlobalStack` prop on `VOverlay` from `false` to `true` during the lifetime of the component, then there is risk that the clean-up code () would not run - an orphan entry would remain on the `globalStack`. By "clean-up" code I mean only this bit:

```js
const idx = globalStack.findIndex(v => v[0] === vm.uid)
globalStack.splice(idx, 1)
```